src/mraa.jl

```diff
@@ -27,15 +27,15 @@ function set_log_level(p)
 end
 
 function platform_name()
-    bytestring(ccall((:mraa_get_platform_name, libmraa), Ptr{Uint8}, ()))
+    String(ccall((:mraa_get_platform_name, libmraa), Ptr{UInt8}, ()))
 end
 
 function set_priority(p)
     ccall((:mraa_set_priority, libmraa), Cint, (Cuint,), p)
 end
 
 function version()
-    VersionNumber(bytestring(ccall((:mraa_get_version, libmraa), Ptr{Uint8}, ())))
+    VersionNumber(String(ccall((:mraa_get_version, libmraa), Ptr{UInt8}, ())))
```

Comment:
If this is done without Compat it may be good to bump REQUIRE to `julia 0.5` as well.